{
 "errors": {
  "google.protobuf": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/__init__.py'",
    ""
   ]
  },
  "google.protobuf.any_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/any_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/any_pb2.py'",
    ""
   ]
  },
  "google.protobuf.any_test_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/any_test_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/any_test_pb2.py'",
    ""
   ]
  },
  "google.protobuf.api_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/api_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/api_pb2.py'",
    ""
   ]
  },
  "google.protobuf.descriptor": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/descriptor.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/descriptor.py'",
    ""
   ]
  },
  "google.protobuf.descriptor_database": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/descriptor_database.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/descriptor_database.py'",
    ""
   ]
  },
  "google.protobuf.descriptor_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/descriptor_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/descriptor_pb2.py'",
    ""
   ]
  },
  "google.protobuf.descriptor_pool": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/descriptor_pool.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/descriptor_pool.py'",
    ""
   ]
  },
  "google.protobuf.duration_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/duration_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/duration_pb2.py'",
    ""
   ]
  },
  "google.protobuf.empty_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/empty_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/empty_pb2.py'",
    ""
   ]
  },
  "google.protobuf.field_mask_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/field_mask_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/field_mask_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/__init__.py'",
    ""
   ]
  },
  "google.protobuf.internal._parameterized": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/_parameterized.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/_parameterized.py'",
    ""
   ]
  },
  "google.protobuf.internal.any_test_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/any_test_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/any_test_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.api_implementation": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/api_implementation.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/api_implementation.py'",
    ""
   ]
  },
  "google.protobuf.internal.containers": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/containers.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/containers.py'",
    ""
   ]
  },
  "google.protobuf.internal.decoder": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/decoder.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/decoder.py'",
    ""
   ]
  },
  "google.protobuf.internal.descriptor_database_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/descriptor_database_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/descriptor_database_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.descriptor_pool_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/descriptor_pool_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/descriptor_pool_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.descriptor_pool_test1_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/descriptor_pool_test1_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/descriptor_pool_test1_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.descriptor_pool_test2_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/descriptor_pool_test2_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/descriptor_pool_test2_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.descriptor_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/descriptor_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/descriptor_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.encoder": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/encoder.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/encoder.py'",
    ""
   ]
  },
  "google.protobuf.internal.enum_type_wrapper": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/enum_type_wrapper.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/enum_type_wrapper.py'",
    ""
   ]
  },
  "google.protobuf.internal.factory_test1_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/factory_test1_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/factory_test1_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.factory_test2_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/factory_test2_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/factory_test2_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.file_options_test_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/file_options_test_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/file_options_test_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.generator_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/generator_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/generator_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.import_test_package": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/import_test_package/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/import_test_package/__init__.py'",
    ""
   ]
  },
  "google.protobuf.internal.import_test_package.inner_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/import_test_package/inner_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/import_test_package/inner_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.import_test_package.outer_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/import_test_package/outer_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/import_test_package/outer_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.json_format_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/json_format_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/json_format_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.message_factory_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/message_factory_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/message_factory_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.message_listener": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/message_listener.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/message_listener.py'",
    ""
   ]
  },
  "google.protobuf.internal.message_set_extensions_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/message_set_extensions_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/message_set_extensions_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.message_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/message_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/message_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.missing_enum_values_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/missing_enum_values_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/missing_enum_values_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.more_extensions_dynamic_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/more_extensions_dynamic_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/more_extensions_dynamic_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.more_extensions_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/more_extensions_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/more_extensions_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.more_messages_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/more_messages_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/more_messages_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.packed_field_test_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/packed_field_test_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/packed_field_test_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.proto_builder_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/proto_builder_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/proto_builder_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.python_message": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/python_message.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/python_message.py'",
    ""
   ]
  },
  "google.protobuf.internal.reflection_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/reflection_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/reflection_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.service_reflection_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/service_reflection_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/service_reflection_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.symbol_database_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/symbol_database_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/symbol_database_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.test_bad_identifiers_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/test_bad_identifiers_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/test_bad_identifiers_pb2.py'",
    ""
   ]
  },
  "google.protobuf.internal.test_util": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/test_util.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/test_util.py'",
    ""
   ]
  },
  "google.protobuf.internal.testing_refleaks": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/testing_refleaks.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/testing_refleaks.py'",
    ""
   ]
  },
  "google.protobuf.internal.text_encoding_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/text_encoding_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/text_encoding_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.text_format_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/text_format_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/text_format_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.type_checkers": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/type_checkers.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/type_checkers.py'",
    ""
   ]
  },
  "google.protobuf.internal.unknown_fields_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/unknown_fields_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/unknown_fields_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.well_known_types": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/well_known_types.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/well_known_types.py'",
    ""
   ]
  },
  "google.protobuf.internal.well_known_types_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/well_known_types_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/well_known_types_test.py'",
    ""
   ]
  },
  "google.protobuf.internal.wire_format": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/wire_format.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/wire_format.py'",
    ""
   ]
  },
  "google.protobuf.internal.wire_format_test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/wire_format_test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/internal/wire_format_test.py'",
    ""
   ]
  },
  "google.protobuf.map_unittest_pb2": {
   "exception": "The subprocess /usr/share/miniconda/envs/test/bin/python was killed. Maybe out of memory?",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/compiled/subprocess/__init__.py\", line 241, in _send",
    "    pickle_dump(data, self._get_process().stdin, PICKLE_PROTOCOL)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/_compatibility.py\", line 42, in pickle_dump",
    "    file.flush()",
    "BrokenPipeError: [Errno 32] Broken pipe",
    "",
    "During handling of the above exception, another exception occurred:",
    "",
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/helpers.py\", line 488, in wrapper",
    "    return func(self, line, column, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 213, in complete",
    "    return completion.complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 169, in complete",
    "    cached_name, completion_names = self._complete_python(leaf)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 310, in _complete_python",
    "    completion_names += self._complete_global_scope()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 375, in _complete_global_scope",
    "    for filter in filters:",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/context.py\", line 495, in get_global_filters",
    "    b = next(base_context.inference_state.builtins_module.get_filters(), None)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/cache.py\", line 44, in wrapper",
    "    rv = function(obj, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/__init__.py\", line 128, in builtins_module",
    "    builtins_module, = self.import_module((module_name,), sys_path=())",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/__init__.py\", line 112, in import_module",
    "    self, import_names, sys_path, prefer_stubs=prefer_stubs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 383, in import_module_by_names",
    "    ) for parent_module_value in value_set",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 383, in <listcomp>",
    "    ) for parent_module_value in value_set",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/__init__.py\", line 21, in wrapper",
    "    return built_functions[public_name](*args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/flask.py\", line 20, in wrapper",
    "    return callback(inference_state, import_names, module_context, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 119, in wrapper",
    "    for p in parent_module_values",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/base_value.py\", line 426, in from_sets",
    "    for set_ in sets:",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 119, in <genexpr>",
    "    for p in parent_module_values",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 415, in import_module",
    "    is_global_search=True,",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/compiled/subprocess/__init__.py\", line 133, in wrapper",
    "    kwargs=kwargs,",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/compiled/subprocess/__init__.py\", line 223, in run",
    "    self._send(inference_state_id, None)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/compiled/subprocess/__init__.py\", line 245, in _send",
    "    % self._executable)",
    "jedi.api.exceptions.InternalError: The subprocess /usr/share/miniconda/envs/test/bin/python was killed. Maybe out of memory?",
    ""
   ]
  },
  "google.protobuf.message": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/message.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/message.py'",
    ""
   ]
  },
  "google.protobuf.proto_builder": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/proto_builder.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/proto_builder.py'",
    ""
   ]
  },
  "google.protobuf.pyext": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/pyext/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/pyext/__init__.py'",
    ""
   ]
  },
  "google.protobuf.pyext.cpp_message": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/pyext/cpp_message.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/pyext/cpp_message.py'",
    ""
   ]
  },
  "google.protobuf.pyext.python_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/pyext/python_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/pyext/python_pb2.py'",
    ""
   ]
  },
  "google.protobuf.reflection": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/reflection.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/reflection.py'",
    ""
   ]
  },
  "google.protobuf.service": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/service.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/service.py'",
    ""
   ]
  },
  "google.protobuf.service_reflection": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/service_reflection.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/service_reflection.py'",
    ""
   ]
  },
  "google.protobuf.struct_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/struct_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/struct_pb2.py'",
    ""
   ]
  },
  "google.protobuf.symbol_database": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/symbol_database.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/symbol_database.py'",
    ""
   ]
  },
  "google.protobuf.test_messages_proto3_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/test_messages_proto3_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/test_messages_proto3_pb2.py'",
    ""
   ]
  },
  "google.protobuf.text_format": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/text_format.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/text_format.py'",
    ""
   ]
  },
  "google.protobuf.timestamp_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/timestamp_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/timestamp_pb2.py'",
    ""
   ]
  },
  "google.protobuf.type_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/type_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/type_pb2.py'",
    ""
   ]
  },
  "google.protobuf.unittest_arena_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_arena_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_arena_pb2.py'",
    ""
   ]
  },
  "google.protobuf.unittest_custom_options_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_custom_options_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_custom_options_pb2.py'",
    ""
   ]
  },
  "google.protobuf.unittest_import_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_import_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_import_pb2.py'",
    ""
   ]
  },
  "google.protobuf.unittest_import_public_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_import_public_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_import_public_pb2.py'",
    ""
   ]
  },
  "google.protobuf.unittest_mset_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_mset_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_mset_pb2.py'",
    ""
   ]
  },
  "google.protobuf.unittest_mset_wire_format_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_mset_wire_format_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_mset_wire_format_pb2.py'",
    ""
   ]
  },
  "google.protobuf.unittest_no_arena_import_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_no_arena_import_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_no_arena_import_pb2.py'",
    ""
   ]
  },
  "google.protobuf.unittest_no_arena_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_no_arena_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_no_arena_pb2.py'",
    ""
   ]
  },
  "google.protobuf.unittest_no_generic_services_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_no_generic_services_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_no_generic_services_pb2.py'",
    ""
   ]
  },
  "google.protobuf.unittest_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/unittest_pb2.py'",
    ""
   ]
  },
  "google.protobuf.wrappers_pb2": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/wrappers_pb2.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpabic429p/lib/python3.5/site-packages/google/protobuf/wrappers_pb2.py'",
    ""
   ]
  }
 },
 "symbols": [
  "google.protobuf.descriptor",
  "google.protobuf.descriptor_pb2",
  "google.protobuf.descriptor_pool",
  "google.protobuf.internal.enum_type_wrapper",
  "google.protobuf.json_format.Error",
  "google.protobuf.json_format.MessageToDict",
  "google.protobuf.json_format.MessageToJson",
  "google.protobuf.json_format.Parse",
  "google.protobuf.json_format.SerializeToJsonError",
  "google.protobuf.json_format._ConvertBool",
  "google.protobuf.json_format._ConvertFloat",
  "google.protobuf.json_format._ConvertInteger",
  "google.protobuf.json_format._ConvertScalarFieldValue",
  "google.protobuf.json_format._CreateMessageFromTypeUrl",
  "google.protobuf.json_format._DuplicateChecker",
  "google.protobuf.json_format._FLOAT_TYPES",
  "google.protobuf.json_format._INFINITY",
  "google.protobuf.json_format._INT64_TYPES",
  "google.protobuf.json_format._INT_OR_FLOAT",
  "google.protobuf.json_format._INT_TYPES",
  "google.protobuf.json_format._IsMapEntry",
  "google.protobuf.json_format._IsWrapperMessage",
  "google.protobuf.json_format._NAN",
  "google.protobuf.json_format._NEG_INFINITY",
  "google.protobuf.json_format._Parser",
  "google.protobuf.json_format._Printer",
  "google.protobuf.json_format._TIMESTAMPFOMAT",
  "google.protobuf.json_format._UNPAIRED_SURROGATE_PATTERN",
  "google.protobuf.json_format._WKTJSONMETHODS",
  "google.protobuf.json_format.__author__",
  "google.protobuf.json_format.descriptor",
  "google.protobuf.json_format.symbol_database",
  "google.protobuf.message",
  "google.protobuf.message_factory.GetMessages",
  "google.protobuf.message_factory.MessageFactory",
  "google.protobuf.message_factory._FACTORY",
  "google.protobuf.message_factory.__author__",
  "google.protobuf.message_factory.descriptor_pool",
  "google.protobuf.message_factory.message",
  "google.protobuf.message_factory.reflection",
  "google.protobuf.reflection",
  "google.protobuf.source_context_pb2.DESCRIPTOR",
  "google.protobuf.source_context_pb2.SourceContext",
  "google.protobuf.source_context_pb2._SOURCECONTEXT",
  "google.protobuf.source_context_pb2._b",
  "google.protobuf.source_context_pb2._sym_db",
  "google.protobuf.source_context_pb2.descriptor",
  "google.protobuf.source_context_pb2.descriptor_pb2",
  "google.protobuf.source_context_pb2.message",
  "google.protobuf.source_context_pb2.reflection",
  "google.protobuf.source_context_pb2.symbol_database",
  "google.protobuf.symbol_database",
  "google.protobuf.text_encoding.CEscape",
  "google.protobuf.text_encoding.CUnescape",
  "google.protobuf.text_encoding._CUNESCAPE_HEX",
  "google.protobuf.text_encoding._cescape_byte_to_str",
  "google.protobuf.text_encoding._cescape_highbit_to_str",
  "google.protobuf.text_encoding._cescape_utf8_to_str",
  "google.protobuf.unittest_import_pb2",
  "google.protobuf.unittest_proto3_arena_pb2.DESCRIPTOR",
  "google.protobuf.unittest_proto3_arena_pb2.FOREIGN_BAR",
  "google.protobuf.unittest_proto3_arena_pb2.FOREIGN_BAZ",
  "google.protobuf.unittest_proto3_arena_pb2.FOREIGN_FOO",
  "google.protobuf.unittest_proto3_arena_pb2.FOREIGN_ZERO",
  "google.protobuf.unittest_proto3_arena_pb2.ForeignEnum",
  "google.protobuf.unittest_proto3_arena_pb2.ForeignMessage",
  "google.protobuf.unittest_proto3_arena_pb2.NestedTestAllTypes",
  "google.protobuf.unittest_proto3_arena_pb2.TestAllTypes",
  "google.protobuf.unittest_proto3_arena_pb2.TestEmptyMessage",
  "google.protobuf.unittest_proto3_arena_pb2.TestPackedTypes",
  "google.protobuf.unittest_proto3_arena_pb2.TestUnpackedTypes",
  "google.protobuf.unittest_proto3_arena_pb2._FOREIGNENUM",
  "google.protobuf.unittest_proto3_arena_pb2._FOREIGNMESSAGE",
  "google.protobuf.unittest_proto3_arena_pb2._NESTEDTESTALLTYPES",
  "google.protobuf.unittest_proto3_arena_pb2._TESTALLTYPES",
  "google.protobuf.unittest_proto3_arena_pb2._TESTALLTYPES_NESTEDENUM",
  "google.protobuf.unittest_proto3_arena_pb2._TESTALLTYPES_NESTEDMESSAGE",
  "google.protobuf.unittest_proto3_arena_pb2._TESTEMPTYMESSAGE",
  "google.protobuf.unittest_proto3_arena_pb2._TESTPACKEDTYPES",
  "google.protobuf.unittest_proto3_arena_pb2._TESTUNPACKEDTYPES",
  "google.protobuf.unittest_proto3_arena_pb2._b",
  "google.protobuf.unittest_proto3_arena_pb2._sym_db",
  "google.protobuf.unittest_proto3_arena_pb2.descriptor",
  "google.protobuf.unittest_proto3_arena_pb2.descriptor_pb2",
  "google.protobuf.unittest_proto3_arena_pb2.enum_type_wrapper",
  "google.protobuf.unittest_proto3_arena_pb2.google_dot_protobuf_dot_unittest__import__public__pb2",
  "google.protobuf.unittest_proto3_arena_pb2.message",
  "google.protobuf.unittest_proto3_arena_pb2.reflection",
  "google.protobuf.unittest_proto3_arena_pb2.symbol_database",
  "google.protobuf.unittest_proto3_arena_pb2.unittest_import_pb2"
 ]
}